goodtablesio/blueprints/github.py

```diff
@@ -1,59 +1,55 @@
+import re
 import os
 import subprocess
 import tempfile
 import uuid
 import logging
-
 from flask import Blueprint, request, abort
-
-from goodtablesio import handlers
-
-
+from .. import helpers
 log = logging.getLogger(__name__)
 
 
-TABULAR_EXTENSIONS = ['csv', 'xls', 'xlsx', 'ods']
+SECRET = 'HUGI#6A2X|e{.Rkn`zg?a!`/9(&(Y7WYQqW#5.(&][s&-W(A8y;85pC:&;H<v*aw'
 CLONE_DIR = '/tmp'
 
 
 github = Blueprint('github', __name__, url_prefix='/github')
 
-SECRET = 'HUGI#6A2X|e{.Rkn`zg?a!`/9(&(Y7WYQqW#5.(&][s&-W(A8y;85pC:&;H<v*aw'
-
 
 @github.route('/hook', methods=['POST'])
 def create_task():
 
     # TODO: check origin with secret
+    # TODO: process errors
 
+    # Get payload
     payload = request.get_json()
     if not payload:
         abort(400)
 
+    # Get task id
     task_id = str(uuid.uuid4())
```

Comment:
The ones created by celery by default have the `str()` form (eg `debb64bb-fee1-46fb-b28f-9d47fd53d270`), not the `hex` one (`debb64bbfee146fbb28f9d47fd53d270`). This way we get consistent ids across integrations.